shopfloor/services/location_content_transfer.py

```diff
@@ -273,6 +280,7 @@ def find_work(self):
         response = self._recover_started_picking()
         if response:
             return response
+        self._actions_for("lock").advisory("find_work")
```

Comment:
Can the advisory lock name move to a variable or method? And I would give it a more unique name like location_content_transfer_find_work